packages/core/android/src/main/kotlin/com/datadog/reactnative/DdSdkNativeInitialization.kt

```diff
@@ -0,0 +1,341 @@
+            configBuilder.useViewTrackingStrategy(ActivityViewTrackingStrategy(false))
+        } else {
+            configBuilder.useViewTrackingStrategy(NoOpViewTrackingStrategy)
+        }
+
+        if (configuration.nativeInteractionTracking == false) {
+            configBuilder.disableUserInteractionTracking()
+        }
+
+        configBuilder.setResourceEventMapper(
+            object : EventMapper<ResourceEvent> {
+                override fun map(event: ResourceEvent): ResourceEvent? {
+                    if (event.context?.additionalProperties?.containsKey(DdSdkImplementation.DD_DROP_RESOURCE) ==
+                        true
+                    ) {
+                        return null
+                    }
+                    return event
+                }
+            }
+        )
+
+        configBuilder.setActionEventMapper(
+            object : EventMapper<ActionEvent> {
+                override fun map(event: ActionEvent): ActionEvent? {
+                    if (event.context?.additionalProperties?.containsKey(DdSdkImplementation.DD_DROP_ACTION) == true
+                    ) {
+                        return null
+                    }
+                    return event
+                }
+            }
+        )
+
+        _RumInternalProxy.setTelemetryConfigurationEventMapper(
+            configBuilder,
+            object : EventMapper<TelemetryConfigurationEvent> {
+                override fun map(
+                    event: TelemetryConfigurationEvent
+                ): TelemetryConfigurationEvent? {
+                    event.telemetry.configuration.trackNativeErrors =
+                        configuration.nativeCrashReportEnabled
+                    // trackCrossPlatformLongTasks will be deprecated for trackLongTask
+                    event.telemetry.configuration.trackCrossPlatformLongTasks =
+                        configuration.longTaskThresholdMs != 0.0
+                    event.telemetry.configuration.trackLongTask =
+                        configuration.longTaskThresholdMs != 0.0
+                    event.telemetry.configuration.trackNativeLongTasks =
+                        configuration.nativeLongTaskThresholdMs != 0.0
+
+                    event.telemetry.configuration.initializationType =
+                        configuration.configurationForTelemetry?.initializationType
+                    event.telemetry.configuration.trackInteractions =
+                        configuration.configurationForTelemetry?.trackInteractions
+                    event.telemetry.configuration.trackErrors =
+                        configuration.configurationForTelemetry?.trackErrors
+                    event.telemetry.configuration.trackResources =
+                        configuration.configurationForTelemetry?.trackNetworkRequests
+                    event.telemetry.configuration.trackNetworkRequests =
+                        configuration.configurationForTelemetry?.trackNetworkRequests
```

Comment:
both are using `trackNetworkRequests`, should it be like that?

```suggestion
                    event.telemetry.configuration.trackResources =
                        configuration.configurationForTelemetry?.trackResources
                    event.telemetry.configuration.trackNetworkRequests =
                        configuration.configurationForTelemetry?.trackNetworkRequests
```

Reply:
For React Native there's only one configuration field for both, so it's expected that they have the same value.
I believe `trackResources` is the legacy version of `trackNetworkRequests` in telemetry since it seems to not be set anymore in the native SDKs.